2022 Jan/Experience of using Docker.md

```diff
@@ -0,0 +1,55 @@
+# Experience of using Docker
+
+Created: January 26, 2022
+Created by: Anonymous
+Tags: Software Tool
+
+# What is Docker?
+
+- One of Container Runtime software tools. It executes and monitors containers.
+
+# Why we use?
+
+- There are lots of benefits while using containers.
+    - Lightweight
+    - Isolation
+    - Portability
+- Dockerfile = record of managing multiple servers, Docker image = fixed server status
+- In terms of TDD, we can make safe server images successfully with many trials and errors.
+- Easily switches port numbers or other environment properties by environment paths.
+
+# Practical Case
+
+## Dockfile
+
+```jsx
```

Comment:
could this be.. shell type?

Reply:
I had figured out that jsx is an extension of javascript. It can be used for React.js. I think I need to look into the correct type.

Reply:
I fixed into `console`

Reply:
GOOOD